libs/utils/trace.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, platform, data_dir, events,
                  plots_prefix=''):
 
         # The platform used to run the experiments
-        self.platform = platform
+        self.platform = platform or {}
```

Comment:
OH.. do you mean actually re-order the parameters to have `platform` at the end? I do agree that would be a better API but I dunno about making such a big breaking change in arguably the most important function in LISA...